README.md

```diff
@@ -790,20 +748,22 @@ It requires a two-step setup:
 
 - Add following dependency to your build:
 ```scala
-libraryDependencies += "com.github.valskalla" %% "odin-slf4j" % "0.11.0"
+libraryDependencies += "com.github.valskalla" %% "odin-slf4j" % "0.11.0+3-19c38e47+20210402-1813-SNAPSHOT"
 ```
 - Create Scala class `ExternalLogger` somewhere in the project:
 ```scala
-import cats.effect.{Clock, Effect, IO}
+import cats.effect.{Sync, IO}
+import cats.effect.std.Dispatcher
+import cats.effect.unsafe.implicits.global
 import io.odin._
 import io.odin.slf4j.OdinLoggerBinder
 
 //effect type should be specified inbefore
 //log line will be recorded right after the call with no suspension
 class ExternalLogger extends OdinLoggerBinder[IO] {
 
-  implicit val F: Effect[IO] = IO.ioEffect
-  implicit val clock: Clock[IO] = Clock.create
+  implicit val F: Sync[IO] = IO.asyncForIO
+  implicit val dispatcher: Dispatcher[IO] = Dispatcher[IO].allocated.unsafeRunSync()._1
```

Comment:
bad idea to do `allocated` on `Dispatcher`, can you use an IORuntime instead?

Reply:
I guess it's not really possible since the interface is supposed to work on any `F[_]`...